### ISSUE TITLE
Errors processing resources with NO Tags

### ISSUE THREAD

**Issue by lyricnz** [Innablr/revolver#321]
Looks like various places assume non-undefined Tags attribute on resources.

```
 TypeError  Cannot read properties of undefined (reading 'find')
error stack:
  • ec2.js      InstrumentedEc2.tag
        /drivers/ec2.js:43
  • powercycleCentral.js        PowerCycleCentralPlugin.generateActions
        /plugins/powercycleCentral.js:69
  • accountRevolver.js
        /lib/accountRevolver.js:90
```

eg:
```javascript
  tag(key: string) {
    const tag = this.resource.Tags.find((xt: Tag) => xt.Key === key);
    if (tag !== undefined) {
      return tag.Value;
    }
  }
```

**Comment by lyricnz** [comment]
Fixed in #323

**Comment by lyricnz** [comment]
The fix was split from #323 into #324
